src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -269,7 +273,20 @@ export const Offer: React.FC<OfferProps> = ({ sessionToken }) => {
               </Formik>
             </>
           )}
-          {data?.quote && <pre>{JSON.stringify(data, null, 2)}</pre>}
+          {data?.quote && (
+            <>
+              <LinkButton
+                background={colorsV3.gray100}
+                foreground={colorsV3.gray900}
+                to={location.pathname.replace('debugger', 'offer')}
```

Comment:
clever! But imo we should "hard-code" the link target if we can. Something like ``to={`/${currentLocale}/new-member/offer`}``

i'm thinking for reasons such as
1) if we'd replace the route with something else, we can't just look for `*/new-member/offer`.
2) if i change the route of `debugger` i'd be surprised why  the link _to_ the offer page would stop working

Reply:
You're right. I'll change it!